corehq/apps/style/templates/style/includes/bootstrap3/ko.html

```diff
@@ -17,5 +17,8 @@
 <script type="text/javascript" src="{% new_static 'style/lib/knockout_plugins/knockout_mapping.ko.min.js' %}"></script>
 <script type="text/javascript" src="{% new_static 'style/ko/global_handlers.ko.js' %}"></script>
 <script type="text/javascript" src="{% new_static 'style/ko/knockout_bindings.ko.js' %}"></script>
+
+<script type="text/javascript" src="{% new_static 'style/ko/components/inline_edit.js' %}"></script>
```

Comment:
Any new components need to be added here, which is annoying. Integrating knockout with RequireJS (or another AMD loader) would let us load these files only as needed.